### ISSUE TITLE
don't work on python3.6 telethon 1.23.0

### ISSUE THREAD

**Issue by firexx** [gentoo-root/telegram-tracker#18]
python3 -m track 'me'
shows :

~2021-09-24 @ 16:15:24: User went offline.

and do nothing if I also go online again.

if I being online the script show also the same message.

**Comment by scarlion1** [comment]
@firexx  Yeah Telegram probably changed the API long ago but this project was abandoned before that.  Have ya found any viable alternatives?

**Comment by Dermalog** [comment]
not really. but you can do it if you write your own telegram client. telegram bot can't do it(imho).

**Comment by firexx** [comment]
> Have ya found any viable alternatives?

not really. but you can do it if you write your own telegram client. telegram bot can't do it(imho).
